app/views/users/activate_account.html.erb

```diff
@@ -10,14 +10,25 @@
 </div>
 
 <%- content_for(:no_ember_head) do %>
+  <%= script "preload_store" %>
```

Comment:
I just needed access to the Discourse object which is not available in the no_ember layout. These changes are the minimum I could find to make that work. There are a lot more differences between the no_ember and application layouts than this, the page would look different if I changed the layout.